MapboxNavigation/NavigationMapView.swift

```diff
@@ -251,6 +258,10 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         
         installUserCourseView()
         showsUserLocation = false
+        
+        if routeLineTracksTraversal, let firstRoute = routes?.first {
```

Comment:
I think I've asked about this before, but can you make sure that this check is really needed here at all? I think `commonInit()` will never have valid `routes` since they're assigned only after calling `show(_:legIndex:)` method. This means that we can remove this part completely.

Reply:
I'll check this part again. Thanks for pointing out this problem!